src/aleph/config.py

```diff
@@ -83,3 +85,10 @@ def get_defaults():
             "traces_sample_rate": None,
         },
     }
+
+
+app_config = Config(schema=get_defaults())
```

Comment:
I see multiple uses of `aleph.config.app_config`. Why not using `aleph.config.get_config()` everywhere and marking this variable as private with a `_` prefix ?

Reply:
Is this variable useful versus using `@lru_cache` on `def get_config()` ?

Reply:
It conveys the intent better IMO. Using LRU cache when there's only one return value seems a bit counterintuitive.

Reply:
What about the first comment ?

Reply:
I think the usage in job_utils.py is legit as we need to initialize the global variable. The other one in `get_ipfs_api` looks like an oversight from my part.